### DISCUSSION TITLE
Error when sending and receiving text messages from two nodes

### DISCUSSION THREAD

**Discussion by ghost** [dtn7/dtn7-rs#31] [Q&A] [answered]
![Error](https://user-images.githubusercontent.com/64239872/181806039-1f2a7169-5944-4d35-88f0-e5addcddd587.PNG)

How do I go about fixing this error?

**Comment by gh0st42** [comment, marked as answer]
This indicates that no dtnd was started on the nodes, thus, the CLI tools get a connection refused when trying to communicate with the local dtnd instance.

As I have no idea what your prior steps were, I can only guess that you forget to start the dtnd or something went wrong while copying the binaries to the system. Instead of using `cea` as suggested in the guide, you can also just start `dtnd` interactively on each node by entering the commands (`dtnd -n $(hostname) -e incoming -C mtcp -r epidemic`) in one of coreemus terminal windows. If you followed the steps of the guide there should also be logs in each of the nodes for dtnd (`nohup.dtnd.*.log`).

For proper help you should also post instructions to exactly reproduce your issue and text is preferred over screeenshots. Also, we have a discussion board in the project where general help can be given. The issues are more for reporting bugs in the code itself, not user support :)
